scene/gui/tree.cpp

```diff
@@ -1765,7 +1842,12 @@ int Tree::propagate_mouse_event(const Point2i &p_pos, int x_ofs, int y_ofs, bool
 		}
 
 		if (!p_item->disable_folding && !hide_folding && !p_item->cells[col].editable && !p_item->cells[col].selectable && p_item->get_children()) {
-			p_item->set_collapsed(!p_item->is_collapsed());
+			// ALT or equivalent is held, change collapse behavior.
+			if (Input::get_singleton()->is_key_pressed(KEY_ALT)) {
```

Comment:
I am not sure hardcoding the ALT key here is a good idea, but I don't know if it is done elsewhere this way. I would rather use a method that allows customization.

Reply:
Thinking twice about it, maybe leave it as it is, when the ALT is forced currently (don't know)

Reply:
I wasn't sure about it either at first, but at the same time, I'm literally coding the "alternate click" function of the little collapse/expand arrow and we already do hardcode the normal click itself.

The few trees in the editor that had a similar function were also hardcoding a key, but some were using ctrl, shift, alt or combination of them to do it.

ALT + Click is definitely the most widespread and standard combination for this kind of behaviour though.